imap_data_access/__init__.py

```diff
@@ -7,9 +7,10 @@
 import os
 from pathlib import Path
 
+from imap_data_access.file_validation import ScienceFilepath
 from imap_data_access.io import download, query, upload
 
-__all__ = ["query", "download", "upload"]
+__all__ = ["query", "download", "upload", "ScienceFilepath"]
```

Comment:
Is this something we want in the top-level namespace? Versus having this imported from the `file_validation` module? I don't have a strong preference either way, so just something to consider.

Reply:
🤷  I don't have a preference either, this is slightly cleaner since file_validation only has the `ScienceFilepath` class in it anyway